internal/testutil/capture.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package testutil
+
+import (
+	"bytes"
+	"io"
+	"os"
+)
+
+// Captures the stdout and stderr of a given function
+func Capture(f func()) (string, string) {
```

Comment:
please pass in an io.Writer as the first argument (and use Fprintf) if you want to test output.

a lot of the ML API tests do this.

Reply:
Thanks, changed